man/man4/zfs.4

```diff
@@ -2177,9 +2177,23 @@ for each I/O submitter.
 When unset, requests are handled asynchronously by a thread pool.
 The number of requests which can be handled concurrently is controlled by
 .Sy zvol_threads .
-.
-.It Sy zvol_threads Ns = Ns Sy 32 Pq uint
-Max number of threads which can handle zvol I/O requests concurrently.
+On blk-mq kernels, zvol_request_sync is ignored.
+.
+.It Sy zvol_threads Ns = Ns Sy 0 Pq uint
+The number of threads to use for processing zvol block IOs.
+On older non-blk-mq kernels, zvol_threads is the total number of threads to
```

Comment:
again, Sy z_t.

also, imo, blk-mq should be Li (here, below, and above) because otherwise it's soup

Reply:
> blk-mq should be Li 

I'm a little confused, are you suggesting it be like this?:
```
On older non-
.Li blk-mq
kernels,
```